curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-getters-and-setters-to-control-access-to-an-object.english.md

```diff
@@ -9,7 +9,7 @@ forumTopicId: 301220
 <section id='description'>
 You can obtain values from an object and set the value of a property within an object.
 These are classically called <dfn>getters</dfn> and <dfn>setters</dfn>.
-Getter functions are meant to simply return (get) the value of an object's private variable to the user without the user directly accessing the private variable.
+Getter functions are meant to access the value of an object's private variable without the user directly accessing the private variable. This value can be returned with or without performing any additional modifications to it.
```

Comment:
>Getter functions are meant to access...

I would change this to:
> Getter functions return...

Reply:
I think that these changes will really help! When I first started learning how to program earlier this year everything was a little bit confusing for me. I think your updates will help clear everything up for people.